scripts/dev/sync-paios-skills.sh

```
@@ -0,0 +1,82 @@
+    echo "[sync] ERROR: missing ${SRC}/${skill}/SKILL.md" >&2
+    exit 1
+  fi
+  echo "[sync]   found: ${skill}"
+done
+
+# --- Claude Code + Cursor (native skills/ directory) ---
+echo "[sync] Syncing to native skills adapters (.claude/skills, .cursor/skills)..."
+for dest in "${NATIVE_DESTS[@]}"; do
+  mkdir -p "${dest}"
+  for skill in "${SKILLS[@]}"; do
+    rm -rf "${dest}/${skill}"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)